doc/user/isisd.rst

```diff
@@ -272,7 +272,7 @@ ISIS interface
 Showing ISIS information
 ========================
 
-.. clicmd:: show isis summary
+.. clicmd:: show isis summary [json]
```

Comment:
Should this be `[summary|json]` instead.

Reply:
No, I think it's clear the initial format because you can have a `summary` command or a `summary json` command.